### ISSUE TITLE
Bottom padding API

### ISSUE THREAD

**Issue by davidseek** [stewwan/capacitor-intercom#3]
Native iOS/Android has API to set the bottom padding of the 
Chat Bubble. In my App the default position overlays navigation elements.

Can you please implement the API to use 

```
iOS: Intercom.setBottomPadding(bottomPadding) 
Android: Intercom.client().setBottomPadding(int bottomPadding)
```

Would be highly appreciated.

https://developers.intercom.com/installing-intercom/docs/ios-configuration
https://developers.intercom.com/installing-intercom/docs/android-configuration

**Comment by stewwan** [comment]
hey @davidseek 
what about

https://www.intercom.com/help/en/articles/1118385-can-i-move-the-messenger-s-position-on-my-product-or-site

**Comment by stewwan** [comment]
I mean, changing it via intercom web interface. I tried once and it worked perfectly.

**Comment by davidseek** [comment]
Yeah. 

But this way you limit yourself down to a single value across
all devices and orientations.

If the API integration is not worth it in your eyes,
then I will go this route, but I prefer to keep web UI and mobile UI
as flexible and separated as possible.

**Comment by davidseek** [comment]
Just take the case iPhone 8 compared to iPhone X.
On iPhone X I need roughly 80p bottom spacing.
On iPhone 8 only 44p. Due to the missing bottom notch.

And I don't see this getting any easier/better long term.
More and more different resolutions and screen options,
especially adding Android will call for a flexible solution.

**Comment by stewwan** [comment]
I would happy add it but this plugin is currently used in my work and we didn't face it, just trying to understand. 

can you set up a minimal app reproduction of problem to show me?

**Comment by davidseek** [comment]
Hey @stewwan 

Sorry, been busy with other parts of the project.
But now the issue is very imminent again.

I have setup this example: https://github.com/davidseek/ionic-intercom-demo

Please add your own keys to `capacitor.config.json`

You will see my problem. 
The bubble is overlaying the tab bar as seen here:

<img width="483" alt="Screen Shot 2019-10-11 at 6 03 46 PM" src="https://user-images.githubusercontent.com/17073950/66692289-da456d80-ec51-11e9-93fa-2743e73868e9.png">

I had tried briefly to adjust the bottom padding as suggested,
but A) it did not apply in the App and B) and even so much more important,
I would have to recode many parts of the web project as the the increased padding on the 
web overlays now many parts of the web page. 

If you could add the natively available bottom padding adjustment option,
it would solve this issue and would be highly appreciated.

I'd help you and perform a pull, 
but I literally have no idea how to create libs for Capacitor.

Appreciate your time very much.

**Comment by stewwan** [comment]
ohh I see

**Comment by davidseek** [comment]
Saw the push. Appreciate it very very much!

**Comment by stewwan** [comment]
shipped in 0.2.0 #5
